src/frontend/utils/test/eøsValidators.test.ts

```diff
@@ -0,0 +1,86 @@
+import type { FeltState } from '@navikt/familie-skjema';
+import { Valideringsstatus } from '@navikt/familie-skjema';
+
+import { erEøsPeriodeGyldig, isEmpty, isNumeric, tellAntallDesimaler } from '../eøsValidators';
+import type { IYearMonthPeriode } from '../kalender';
+import { nyYearMonthPeriode } from '../kalender';
+
+describe('utils/eøsValidators', () => {
+    const nyFeltState = <T>(verdi: T): FeltState<T> => ({
+        feilmelding: '',
+        valider: (feltState, _) => feltState,
+        valideringsstatus: Valideringsstatus.IKKE_VALIDERT,
+        verdi,
+    });
+
+    test('erEøsPeriodeGyldig skal kaste feilmelding hvis fom dato ikke er utfylt', () => {
+        const eøsPeriode: FeltState<IYearMonthPeriode> = nyFeltState(
+            nyYearMonthPeriode(undefined, '2010-05')
+        );
+
+        const valideringsresultat = erEøsPeriodeGyldig(eøsPeriode);
+
+        expect(valideringsresultat.valideringsstatus).toEqual(Valideringsstatus.FEIL);
+        expect(valideringsresultat.feilmelding).toEqual('Fra og med måned må være utfylt');
+    });
+
+    test('erEøsPeriodeGyldig skal kaste feilmelding hvis fom dato er mer enn 1mnd senere enn dagens dato', () => {
+        const eøsPeriode: FeltState<IYearMonthPeriode> = nyFeltState(
+            nyYearMonthPeriode('2099-12', '2100-05')
+        );
+
+        const valideringsresultat = erEøsPeriodeGyldig(eøsPeriode);
+
+        expect(valideringsresultat.valideringsstatus).toEqual(Valideringsstatus.FEIL);
+        expect(valideringsresultat.feilmelding).toEqual(
+            'Du kan ikke sette fra og med (f.o.m.) til måneden etter neste måned eller senere'
+        );
+    });
+
+    test('erEøsPeriodeGyldig skal kaste feilmelding hvis tom dato er mer enn 1mnd senere enn dagens dato', () => {
+        const eøsPeriode: FeltState<IYearMonthPeriode> = nyFeltState(
+            nyYearMonthPeriode('2010-12', '2100-05')
+        );
+
+        const valideringsresultat = erEøsPeriodeGyldig(eøsPeriode);
+
+        expect(valideringsresultat.valideringsstatus).toEqual(Valideringsstatus.FEIL);
+        expect(valideringsresultat.feilmelding).toEqual(
+            'Du kan ikke sette til og med (t.o.m.) til neste måned eller senere'
+        );
+    });
+
+    test('erEøsPeriodeGyldig skal kaste feilmelding hvis fom dato er lengre fram i tid enn initielFom', () => {
```

Comment:
Her trur eg det er rom for ein ekstra test - ein som testar det som namnet indikerer, og ein som testar det som per no er i expect-en, som det står no synes eg det er litt forvirrande.

Reply:
Ah, good catch. Her har jeg blingset på testnavnet 🙈